src/components/molecules/Settings/ProjectList/ProjectList/index.tsx

```diff
@@ -1,110 +1,48 @@
 import Loading from "@reearth/components/atoms/Loading";
-import Text from "@reearth/components/atoms/Text";
 import ProjectCell, {
   Project as ProjectType,
 } from "@reearth/components/molecules/Settings/ProjectList/ProjectCell";
-import { styled, useTheme } from "@reearth/theme";
+import { styled } from "@reearth/theme";
 import { metricsSizes } from "@reearth/theme/metrics";
 
 export type Project = ProjectType;
 
 export type Props = {
   title?: string;
-  archived?: boolean;
   projects?: Project[];
   loading?: boolean;
   onProjectSelect?: (project: Project) => void;
-  onCreationButtonClick?: () => void;
 };
 
-const ProjectList: React.FC<Props> = ({
-  loading,
-  projects,
-  title,
-  archived,
-  onProjectSelect,
-  onCreationButtonClick,
-}) => {
-  const intl = useIntl();
-  const [open, setOpen] = useState(true);
-
-  useEffect(() => {
-    if (archived) {
-      setOpen(false);
-    }
-  }, [archived]);
-  const theme = useTheme();
-
+const ProjectList: React.FC<Props> = ({ loading, projects, onProjectSelect }) => {
   return (
     <>
-      {archived && (
-        <StyledFlex align="center" onClick={() => setOpen(!open)}>
-          <StyledIcon icon="arrowToggle" size={15} color={theme.main.text} />
-          <Text
-            size="m"
-            weight="normal"
-            color={theme.main.text}
-            otherProperties={{ margin: "12px 0" }}>
-            {`${intl.formatMessage({ defaultMessage: "Archived Projects" })} (${
-              projects?.length || 0
-            })`}
-          </Text>
-        </StyledFlex>
-      )}
-      {!archived && (
-        <StyledFlex justify="space-between" align="center">
-          <Text size="m" weight="normal" color={theme.main.text}>
-            {title ||
-              `${intl.formatMessage({ defaultMessage: "Current Projects" })} (${
-                projects?.length || 0
-              })`}
-          </Text>
-          <Button
-            large
-            buttonType="secondary"
-            text={intl.formatMessage({ defaultMessage: "New Project" })}
-            onClick={onCreationButtonClick}
-          />
-        </StyledFlex>
-      )}
       {loading ? (
         <div>
           <Loading />
         </div>
       ) : (
         <>
-          {open && (
-            <ProjectListContainner>
-              {projects?.map(project => (
-                <ProjectCell project={project} key={project.id} onSelect={onProjectSelect} />
-              ))}
-            </ProjectListContainner>
-          )}
+          <ProjectListContainner>
+            {projects?.map(project => (
+              <ProjectCell project={project} key={project.id} onSelect={onProjectSelect} />
+            ))}
+          </ProjectListContainner>
         </>
       )}
     </>
   );
 };
 
-const StyledIcon = styled(Icon)`
-  margin-right: 5px;
-`;
-
 const ProjectListContainner = styled.div`
+  > * {
+    margin-top: ${metricsSizes["4xl"]}px;
+  }
```

Comment:
Move this below with the margin-bottom since they both are being applied to the underlying children components